src/tasks/publish/index.ts

```diff
@@ -67,7 +76,8 @@ export function setupPublishTask(): void {
             managerAddress: params.managerAddress,
             ipfsApiUrl: params.ipfsApiUrl,
             onlyContent: params.onlyContent,
-            noVerify: params.noVerify
+            noVerify: params.noVerify,
+            dryRun: params.dryRun
```

Comment:
Brief aside, but we should be able to have prettier add ending commas in cases like this, to decrease the line diffs.

Reply:
That's a great reason to have trailing commas, thanks for the suggestion!

Reply:
Todo in https://github.com/aragon/buidler-aragon/issues/123